e2e/cypress/integration/system_console/reporting/site_statistics_spec.js

```diff
@@ -245,5 +243,8 @@ describe('System Console > Site Statistics', () => {
                 });
             });
         });
+
+        // # Reset locale
+        cy.apiPatchMe({locale: 'en'});
```

Comment:
This needs to be added in the after block because if the testcase fails in between, this step might not be executed and all subsequent steps will start to fail.

Reply:
Moved the reset to afterEach instead since more tests can be added after this test cases which can be affected if it fails prematurely.